en/simulation/README.md

```diff
@@ -57,13 +57,14 @@ Message | Direction | Description
 
 ## Default PX4 MAVLink UDP Ports
 
-By default, PX4 uses commonly established UDP ports for MAVLink communication with ground control stations (e.g. *QGroundControl*), Offboard APIs (e.g. MAVSDK, MAVROS) and simulator APIs (e.g. Gazebo). These ports are:
+By default, PX4 uses commonly established UDP ports for MAVLink communication with ground control stations (e.g. *QGroundControl*), Offboard APIs (e.g. MAVSDK, MAVROS) and simulator APIs (e.g. Gazebo).
+These ports are:
 
 * UDP Port **14540** is used for communication with offboard APIs.
   Offboard APIs are expected to listen for connections on this port.
 * UDP Port **14550** is used for communication with ground control stations.
   GCS are expected to listen for connections on this port. *QGroundControl* listens to this port by default.
-* TCP Port **4560** is used for communication with simulators.
+* The simulator's local TCP Port **4560** is used for communication with PX4.
```

Comment:
@Stifael Is the line below correct? 

I understand how the UDP stuff works - PX4 broadcasts to local UDP ports, and simulators listening on that port and connect when they detect . Not so sure on TCP - I think the simulator defines the connection by broadcasting to this port and PX4 is expected to listen for the simulator ... but it could be the other way (simulator listens on its port and PX4 broadcasts).

Reply:
The simulator is expected to have a local TCP port of 4560 and PX4 will connect. This basically means that the simulator is just the server that you can reach via 4560 port (similar to http with port 80)

Reply:
Thanks. Stuff I should know, but I don't want to screw up.